<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>913</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Colors</string>
    </property>
    <widget class="QPushButton" name="ledColor0">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>86</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ledColor1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>86</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ledColor2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>86</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ledColor3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>86</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="ledModes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ledModeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LED Lighting Mode</string>
    </property>
   </widget>
   <widget class="QSlider" name="ledSpeed">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>130</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TickPosition::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSlider" name="ledBrightness">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TickPosition::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="ledBrightnessLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QLabel" name="ledSpeedLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyLEDSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="text">
     <string>Apply Settings</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledDirection">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse Direction</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Profiles</string>
    </property>
    <widget class="QPushButton" name="newProfileButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="text">
      <string>New Profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteProfileButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="text">
      <string>Delete Profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveProfileButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="text">
      <string>Save Profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadProfileButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="text">
      <string>Load Profile</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="profileNameInput">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="profileList">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>181</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="profileInputLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile Name</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
